src/seedu/addressbook/commands/Command.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Represents an executable command.
  */
-public class Command {
+public abstract class Command {
```

Comment:
Do we really need to make Command abstract here?

Reply:
Shouldn't we though? It helps dummy proof our code. After all, instantiating a Command object with no actual implementation of functionality is rather useless.

Reply:
Well yes, that's certainly true. In that case though the execute method could be removed as it doesn't serve any purpose..